internal/docsgen/sdkgen.go

```diff
@@ -39,7 +40,9 @@ func GenerateContent(ctx context.Context, inputLangs []string, customerID, schem
 		}
 	}
 
-	fmt.Printf("Generating SDK Docs for langs %s...\n", strings.Join(langs, ", "))
+	logger := log.From(ctx)
+
+	logger.WithStyle(styles.Info).Printf("Generating SDK Docs for langs %s...\n", strings.Join(langs, ", "))
```

Comment:
I'm thinking we may want to log some of these directly through `logger.Info` instead. But I'm not sure